src/inmanta/data/__init__.py

```diff
@@ -4634,6 +4634,43 @@ async def get_status_for(
         out = await cls.select_query(query, [env, model_version, rids], no_obj=True)
         return {ResourceIdStr(r["resource_id"]): ResourceState[r["status"]] for r in out}
 
+    @classmethod
+    async def get_current_resource_state(
```

Comment:
this could use some documentation. 

I don't understand why a 'get_current` also gets a model version as an argument?